Makefile

```diff
@@ -40,7 +40,7 @@ coverage: test
 	@echo "Generating coverage report..."
 	@go tool cover -html=coverage.out
 
-build:
+build: tidy
```

Comment:
You can remove this.

Reply:
ack!

Reply:
Resolved: https://github.com/goto/guardian/pull/187/commits/0ce8516d5f5d1ba56a108a0c617294b80d1f0aaf